src/main/java/com/ericsson/gerrit/plugins/eiffel/events/generators/EiffelSourceChangeCreatedEventGenerator.java

```diff
@@ -22,11 +22,15 @@
 import com.google.gerrit.server.data.PatchSetAttribute;
 import com.google.gerrit.server.events.PatchSetCreatedEvent;
 
-public class EiffelSourceChangeCreatedEventGenerator extends EiffelEventGenerator {
+public final class EiffelSourceChangeCreatedEventGenerator extends EiffelEventGenerator {
 
     private static final String TYPE = "EiffelSourceChangeCreatedEvent";
     private static final String TRACKER = "Gerrit";
 
+    static {
```

Comment:
To add unused and unnecessary code just to get more test coverage me not like.

Reply:
I agree with Anders, if the sole purpose of the static block is increased code coverage then it should be avoided. Otherwise, at least, its (frivolous) use must be explain in the comment.

Reply:
Since no one else has given any counter input I'll remove it.